Pages/Signup/CSS/style.css

```diff
@@ -0,0 +1,53 @@
+/* lOGIN-SIGNUP-OUTER-CONTAINER */
+
+.login-signup-outer-container {
+  box-shadow: 5px 5px 5px 0 #00000070;
```

Comment:
could have used rbga instead of hex color

Reply:
done

Reply:
I thought hex are loaded faster than other color formats including rgba. Is it not the case?